packages/flutter_tools/lib/src/project.dart

```diff
@@ -229,38 +229,64 @@ class FlutterProject {
     return manifest;
   }
 
+  /// Re-generates project files necessary to make Gradle builds work on Android
```

Comment:
Is this comment misleading now? it also re-generates for desktop+web projects if they exist

Reply:
Good call, reworded.